### ISSUE TITLE
GLPI Groups

### ISSUE THREAD

**Issue by Kh4os-afk** [glpi-project/glpi#14720]
### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.6

### Bug description

Every ticket assigned to a specific group when it is updated to the "new" status is visible to all other groups created, even if I have no relationship

<img width="769" alt="image" src="https://github.com/glpi-project/glpi/assets/76918680/df581b90-45b3-4ce2-9563-d0d66be70208">

### Your GLPI setup information

`
GLPI 10.0.6 (/helpdesk => C:\xampp\htdocs\helpdesk) Installation mode: TARBALL Current language:pt_BR
`

```
Operating system: Windows NT LARAVEL 10.0 build 14393 (Windows Server 2016) AMD64
PHP 8.1.12 apache2handler (Core, PDO, PDO_OCI, Phar, Reflection, SPL, SimpleXML, apache2handler, bcmath, bz2, calendar, ctype,
	curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, intl, json, ldap, libxml, mbstring, mysqli, mysqlnd,
	oci8, openssl, pcre, pdo_mysql, pdo_sqlite, readline, session, sodium, standard, tokenizer, xml, xmlreader, xmlwriter, zip,
	zlib)
Setup: max_execution_time="120" memory_limit="512M" post_max_size="40M" safe_mode="" session.save_handler="files"
	upload_max_filesize="40M" 
Software: Apache/2.4.54 (Win64) OpenSSL/1.1.1p PHP/8.1.12 (Apache/2.4.54 (Win64) OpenSSL/1.1.1p PHP/8.1.12 Server at
	172.22.22.172 Port 80
)
	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36
Server Software: mariadb.org binary distribution
	Server Version: 10.4.27-MariaDB
	Server SQL Mode: NO_ZERO_IN_DATE,NO_ZERO_DATE,NO_ENGINE_SUBSTITUTION
	Parameters: root@localhost/helpdesk
	Host info: localhost via TCP/IP
	
PHP version (8.1.12) is supported.PHP version (8.1.12) is supported.
Sessions configuration is OK.Sessions configuration is OK.
Allocated memory is sufficient.Allocated memory is sufficient.
mysqli extension is installed.mysqli extension is installed.
Following extensions are installed: dom, fileinfo, json, simplexml.Following extensions are installed: dom, fileinfo, json, simplexml.
curl extension is installed.curl extension is installed.
gd extension is installed.gd extension is installed.
intl extension is installed.intl extension is installed.
libxml extension is installed.libxml extension is installed.
zlib extension is installed.zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (10.4.27) is supported.Database engine version (10.4.27) is supported.
No files from previous GLPI version detected.No files from previous GLPI version detected.
The log file has been created successfully.The log file has been created successfully.
Write access to C:\xampp\htdocs\helpdesk/files/_cache has been validated. Write access to C:\xampp\htdocs\helpdesk/config has been validated. Write access to C:\xampp\htdocs\helpdesk/files/_cron has been validated. Write access to C:\xampp\htdocs\helpdesk/files has been validated. Write access to C:\xampp\htdocs\helpdesk/files/_dumps has been validated. Write access to C:\xampp\htdocs\helpdesk/files/_graphs has been validated. Write access to C:\xampp\htdocs\helpdesk/files/_lock has been validated. Write access to C:\xampp\htdocs\helpdesk/files/_pictures has been validated. Write access to C:\xampp\htdocs\helpdesk/files/_plugins has been validated. Write access to C:\xampp\htdocs\helpdesk/files/_rss has been validated. Write access to C:\xampp\htdocs\helpdesk/files/_sessions has been validated. Write access to C:\xampp\htdocs\helpdesk/files/_tmp has been validated. Write access to C:\xampp\htdocs\helpdesk/files/_uploads has been validated.Write access to C:\xampp\htdocs\helpdesk/files/_cache has been validated.
Write access to C:\xampp\htdocs\helpdesk/config has been validated.
Write access to C:\xampp\htdocs\helpdesk/files/_cron has been validated.
Write access to C:\xampp\htdocs\helpdesk/files has been validated.
Write access to C:\xampp\htdocs\helpdesk/files/_dumps has been validated.
Write access to C:\xampp\htdocs\helpdesk/files/_graphs has been validated.
Write access to C:\xampp\htdocs\helpdesk/files/_lock has been validated.
Write access to C:\xampp\htdocs\helpdesk/files/_pictures has been validated.
Write access to C:\xampp\htdocs\helpdesk/files/_plugins has been validated.
Write access to C:\xampp\htdocs\helpdesk/files/_rss has been validated.
Write access to C:\xampp\htdocs\helpdesk/files/_sessions has been validated.
Write access to C:\xampp\htdocs\helpdesk/files/_tmp has been validated.
Write access to C:\xampp\htdocs\helpdesk/files/_uploads has been validated.
The following directories should be placed outside "C:\xampp\htdocs\helpdesk": ‣ "C:\xampp\htdocs\helpdesk/files" ("GLPI_VAR_DIR") ‣ "C:\xampp\htdocs\helpdesk\config" ("GLPI_CONFIG_DIR") You can ignore this suggestion if you are certain that these directories are not accessible through your web server.The following directories should be placed outside "C:\xampp\htdocs\helpdesk":
‣ "C:\xampp\htdocs\helpdesk/files" ("GLPI_VAR_DIR")
‣ "C:\xampp\htdocs\helpdesk\config" ("GLPI_CONFIG_DIR")
You can ignore this suggestion if you are certain that these directories are not accessible through your web server.
PHP directive "session.cookie_httponly" should be set to "on" to prevent client-side script to access cookie values.PHP directive "session.cookie_httponly" should be set to "on" to prevent client-side script to access cookie values.
exif extension is installed.exif extension is installed.
ldap extension is installed.ldap extension is installed.
openssl extension is installed.openssl extension is installed.
zip extension is installed.zip extension is installed.
bz2 extension is installed.bz2 extension is installed.
Zend OPcache extension is not present.Zend OPcache extension is not present.
Following extensions are installed: ctype, iconv, mbstring, sodium.Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to C:\xampp\htdocs\helpdesk/marketplace has been validated.Write access to C:\xampp\htdocs\helpdesk/marketplace has been validated.
Timezones seems not loaded, see https://glpi-install.readthedocs.io/en/latest/timezones.html.Timezones seems not loaded, see https://glpi-install.readthedocs.io/en/latest/timezones.html.
```

**Comment by cedric-anne** [comment]
Hi,

A `new` ticket will be visible for any user that has the `assign` right. Limiting ability to view "new" tickets will be possible soon (see #14747).

Can you confirm that only users with `assign` right can see the ticket?

**Comment by Kh4os-afk** [comment]
> 

Hello Cedric!
Of course, but a user without `assign` right cannot assign a ticket to itself unlike version 9.5

How can I apply changes from this ticket to my glpi?

**Comment by cedric-anne** [comment]
> Of course, but a user without `assign` right cannot assign a ticket to itself unlike version 9.5

I do not remember if we voluntarly changed something here or if there is a bug.

> How can I apply changes from this ticket to my glpi?

This development has not been validated yet, you have to wait a bit.

**Comment by Kh4os-afk** [comment]
> > Claro, mas um usuário sem `assign`direito não pode atribuir um ticket a si mesmo, ao contrário da versão 9.5
> 
> Não me lembro se mudamos algo voluntariamente aqui ou se há um bug.
> 
> > Como posso aplicar as alterações deste ticket ao meu glpi?
> 
> Este desenvolvimento ainda não foi validado, você tem que esperar um pouco.

Cedric please help us :(
We really need to make the ticket visible only to preference groups with status of "`new`"

**Comment by cedric-anne** [comment]
Hi,

> Of course, but a user without assign right cannot assign a ticket to itself unlike version 9.5

Have you checked #14000?

**Comment by Kh4os-afk** [comment]
> Hi,
> 
> > Of course, but a user without assign right cannot assign a ticket to itself unlike version 9.5
> 
> Have you checked #14000 (fixed by #14435)?

Credic

this worked perfectly 
god bless you

**Comment by cedric-anne** [comment]
I close this issue as it is not a bug. Next GLPI version will enhance this part with a new right dedicated to "new" tickets visibility.
